trunk/NDHMS/Run/hydro.namelist

```diff
@@ -0,0 +1,265 @@
+&HYDRO_nlist
```

Comment:
There's a template hydro.namelist in trunk/NDHMS/template/HYDRO/hydro.namelist.  This should probably be updated there.  The Run directory is created during the build process and doesn't exist in the source code.